windows/README_MINGW

```diff
@@ -249,6 +268,11 @@ cp C:\msys64\mingw64\bin\libgcc_s_sjlj-1.dll `
   , C:\msys64\mingw64\bin\libwinpthread-1.dll `
   C:\Windows
 
+- Copy all dll files from C:\msys64\opt\jack\lib\jack to C:\Windows\jack, these
+are dependencies for libjack:
+
+cp C:\msys64\opt\jack\lib\jack\*.dll C:\Windows\jack
```

Comment:
This was missing in the original contribution. When following all other copy / paste operations I received
```
Drivers/internals found in : C:\WINDOWS
Drivers not found in "C:\WINDOWS\jack": "C:\WINDOWS\jack/*.dll"
Driver folder not found
Could not find any drivers in driver directory!
Failed to create server object
```

After placing all build DLLs in the relative jack path under `C:\Windows`  I could run the server successfully. However, I don't know if this is the best solution or all of the DLLs are really required. Here is a list:
`audioadapter.dll, inprocess.dll, jack_dummy.dll, jack_loopback.dll, jack_net.dll, jack_netone.dll, jack_portaudio.dll, jack_proxy.dll, jack_winmme.dll, netadapter.dll, netmanager.dll, profiler.dll`